opendata-python/opendata/models.py

```diff
@@ -74,6 +75,10 @@ def activities(self):
 
         return self.activities_generator()
 
+    def download_remote_data(self):
+        a = RemoteAthlete(self.id)
+        a.store_locally()
```

Comment:
By the way: This is the only possible blocker for me. The rest we can fix/improve in later releases. I this beta stage I want to release often.

Reply:
Does seem to work - it simply fetches all data and overrides them on a disk. I don't have a test for it, because it would require fetching the actual data... or diving into clever mocking